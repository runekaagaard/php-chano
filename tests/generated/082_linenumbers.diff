001- 01. x
001+ Fatal error: Call to undefined method DtlIter::linenumbers() in /home/rune/workspace/php-misc/php-dtl-iterator/tests/django-generated/082_linenumbers.php on line 13
002- 02. x
003- 03. x
004- 04. x
005- 05. x
006- 06. x
007- 07. x
008- 08. x
009- 09. x
010- 10. x