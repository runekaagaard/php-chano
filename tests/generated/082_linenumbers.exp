01. x
02. x
03. x
04. x
05. x
06. x
07. x
08. x
09. x
10. x