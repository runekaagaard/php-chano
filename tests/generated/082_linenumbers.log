
---- EXPECTED OUTPUT
01. x
02. x
03. x
04. x
05. x
06. x
07. x
08. x
09. x
10. x
---- ACTUAL OUTPUT
Fatal error: Call to undefined method DtlIter::linenumbers() in /home/rune/workspace/php-misc/php-dtl-iterator/tests/generated/082_linenumbers.php on line 13
---- FAILED
