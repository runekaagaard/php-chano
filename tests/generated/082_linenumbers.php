<?php
include dirname(__FILE__) . '/../../DtlIter.php';
$items = new DtlIter(array(array('input' => 'x
x
x
x
x
x
x
x
x
x')));
foreach ($items as $i) echo $i->input->linenumbers();
