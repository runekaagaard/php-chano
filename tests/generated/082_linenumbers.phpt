--TEST--
A generated testfile for the "linenumbers" filter.
--FILE--
<?php
include dirname(__FILE__) . '/../../DtlIter.php';
$items = new DtlIter(array(array('input' => 'x
x
x
x
x
x
x
x
x
x')));
foreach ($items as $i) echo $i->input->linenumbers();
--EXPECT--
01. x
02. x
03. x
04. x
05. x
06. x
07. x
08. x
09. x
10. x