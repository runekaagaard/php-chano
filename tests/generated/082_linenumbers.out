Fatal error: Call to undefined method DtlIter::linenumbers() in /home/rune/workspace/php-misc/php-dtl-iterator/tests/django-generated/082_linenumbers.php on line 13